mibidata/tiff.py

```diff
@@ -64,30 +64,60 @@ def write(filename, image, sed=None, optical=None, ranges=None,
         multichannel: Boolean for whether to create a single multi-channel TIFF,
             or a folder of single-channel TIFFs. Defaults to True; if False,
             the sed and optical options are ignored.
-        write_float: If True, saves the image data as float32 values (for
-            opening properly in certain software such as Halo). Defaults to
-            False which will save the image data as uint16. Note: setting
-            write_float to True does not normalize or scale the data before
-            saving, however saves the integer counts as floating point numbers.
+        dtype: Forces the image data saved as either float or uint16. Can
+            specify or ``np.float32`` to force data to be saved as floating
+            point values or ``np.uint16`` to save as unsigned 16-bit integer
+            values. Defaults to None, which saves data as its original type, if
+            the original type can converted without a loss of data.
```

Comment:
This wording still needs to be improved. I'd suggest

dtype: One of `(np.float32, np.uint16)` to force the the dtype of the saved image data.  Defaults to `None`, which chooses the format based on the data's input type, and will convert to `np.float32` or `np.uint16` from other float or int types, respectively, if it can do so without a loss of data.

Reply:
Done.